frontend/src/modules/machinery/components/Machinery.css

```diff
@@ -23,6 +23,12 @@
     top: 16px;
 }
 
+.machinery > .search-wrapper > label > button {
+    background-color: transparent;
+    border: none;
+    color: #aaaaaa;
```

Comment:
Seems like `<button>` gets some padding assigned (although I don't know from where?), which moves the times icon towards left compared to the magnifier icon.

Adding `padding: 0 4px;` here should fix the unexpected jump.